lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1363,44 +1363,11 @@ input[name="subject"] {
   }
 }
 
-.ecommerce-wrapper{
-  h2{
-    height: 26px;
-    line-height: 26px;
-    span{
-      @include float(right);
-      font-size: 16px;
-      font-weight: bold;
-      span{
-        background: #ddd;
-        padding: 2px 9px;
-        border-radius: 2px;
-        float: none;
-        font-weight: 400;
-      }
-    }
-  }
-  span.tip{
-    display: block;
-    padding: 10px 15px;
-    border-top: 1px solid #ddd;
-    border-bottom: 1px solid #ddd;
-    background: #f8f4ec;
-    color: #3c3c3c;
-    line-height: 30px;
-    .add{
-      @include button(simple, $blue);
-      @extend .button-reset;
-      font-size: em(13);
-      @include float(right);
-    }
-  }
-
-}
```

Comment:
This one is a little different from the other as it's inside another selector. I think both are needed. Can you add it back in?